### PR TITLE
Feat/add unit testing

### PR DESCRIPTION
**Pull request by abbesAlexandre**
test: adding tests for plugins files : nodescure.ts, scorecard.ts

### CODE REVIEW

File: test/scorecard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+// Import Node.js Dependencies
+import { test, after, describe, it } from "node:test";
+import assert from "node:assert";
+
+// Import Third-party Dependencies
+import { execute } from "../src/plugins/scorecard";
+
+
+import { MockAgent, setGlobalDispatcher } from "undici";
```
Comment (PierreDemailly) [inline test/scorecard.spec.ts]:
```suggestion
// Import Third-party Dependencies
import { MockAgent, setGlobalDispatcher } from "undici";

// Import Internal Dependencies
import { execute } from "../src/plugins/scorecard";
```
```

Comment (PierreDemailly) [reply test/scorecard.spec.ts]: Same in `nodesecure.spec.ts`

File: test/nodesecure.spec.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { test, after, describe, it } from "node:test";
```
Comment (PierreDemailly) [inline test/nodesecure.spec.ts]:
```suggestion
// Import Node.js Dependencies
import { test, after, describe, it } from "node:test";
```

File: test/nodesecure.spec.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+  const mockPool = mockAgent.get(kApiUrl);
+
+  setGlobalDispatcher(mockAgent);
+
+  after(async() => {
+    await mockAgent.close();
+  });
+
+  test("should return void", async() => {
```
Comment (PierreDemailly) [inline test/nodesecure.spec.ts]:
```suggestion
  it("should return void", async() => {
```

I think we can use only `it` and remove `test` from the imports ?

File: test/nodesecure.spec.ts
Comment (PierreDemailly) [inline test/nodesecure.spec.ts:1]: Maybe we can move theses to files to `/test/plugins/` ?

**Review by PierreDemailly — CHANGES_REQUESTED** [review]

**Review by PierreDemailly — APPROVED** [review]

**Review by fraxken — APPROVED** [review]

**Pull request merged**